### DISCUSSION TITLE
Could not create paste: Error saving paste. Sorry.

### DISCUSSION THREAD

**Discussion by Litan7** [PrivateBin/PrivateBin#1184] [Q&A] [answered]
### Describe the problem/question

Docker env with conf.php, I am getting error message "Could not create paste: Error saving paste. Sorry."

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

1. I followed the installation guide install docker, nginx, set up DNS, Let's Encrypt 
2. it looks correct when loading https://bin.luxgroup.io but I got "Could not create paste: Error saving paste. Sorry."
3. Found the following error in docker logs.
2023/10/18 11:38:03 [error] 19#19: *25 FastCGI sent in stderr: "PHP message: failed to store the server salt, delete tokens, traffic limiter and user icons won't work; PHP message: failed to store the traffic limiter, it probably contains outdated information; PHP message: failed to store the purge limiter, skipping purge cycle to avoid getting stuck in a purge loop; PHP message: PHP Warning:  mkdir(): Permission denied in /srv/lib/Data/Filesystem.php on line 97" while reading response header from upstream, client: 172.17.0.1, server: , request: "POST / HTTP/1.0", upstream: "fastcgi://unix:/run/php-fpm.sock:", host: "bin.luxgroup.io"
172.17.0.1 - - [18/Oct/2023:11:38:03 +0000] "POST / HTTP/1.0" 200 51 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36" "157.211.x.x"
...

### Server OS

Ubuntu Lightsail

### PrivateBin version

1.6.0

### Browser and version

Vivaldi 6.2.3105.58 (Stable channel) (64-bit) 

### Local operating system and version

Windows 11

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by elrido** [comment, marked as answer]
The message "mkdir: permission denied" is your hot lead. You must ensure that your data volume is owned by the user ID used to run the services. As the [README.md](https://github.com/PrivateBin/docker-nginx-fpm-alpine#running-the-image) describes:

> Note that the volume mounted must be owned by UID 65534 / GID 82. If you run the container in a docker instance with "userns-remap" you need to add your subuid/subgid range to these numbers.

**Reply by Litan7** [reply]
Thank you so so much, I am new to Docker so missed a few things.
I'd like to share how I fixed the issue for those who may run into same issue.

I just ran the following :
sudo chown -R 65534:82 privatebin-data/
docker container restart (docker-ID)
